lib/similarity_utils.py

```diff
@@ -262,3 +263,17 @@ def similarity_one_vs_all_new(data: pd.DataFrame, main_actor: List[Any], reduced
     for actor in actors:
         similarities.append(similarity_new(main_actor, actor, reduced_dataset))
     return list(data["nconst"]), similarities
+
+def get_actor_data(train_data: pd.DataFrame, validate_data: pd.DataFrame, actor_id: str) -> List[Any]:
```

Comment:
w sumie jest już metoda `find_actor`, która szuka aktora w jednym zbiorze danych. Jeżeli robimy tutaj funkcję która szuka aktora dla dwóch zbiorów danych, to zmieńmy może nazwę funkcji na np. `find_actor_two_datasets` i zmieńmy nazwy nazwy dataframów na `data1` i `data2` albo `dataFrame1` i `dataFrame2`. Wtedy w przypadku gdy nie znajdziemy aktora, to możemy printować tekst "W żadnym z tych dwóch zbiorów nie ma wybranego aktora"

Reply:
zmieniłam